Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -174,6 +174,9 @@ extension Docc {
         @available(*, deprecated, message: "Doxygen support is now enabled by default.")
         public var experimentalParseDoxygenCommands = false
 
+        @Flag(help: "Allow catalog directories without the `.docc` extension.")
+        var allowArbitraryCatalogDirectories = false
```

Comment:
I _think_ that we'd want to label this "experimental" until it's been discussed in the forums.

Reply:
To clarify; I don't it needs to be discussed very formally but we should at least have some post that pitches adding this flag since we don't expect developers to monitor PRs